repositories.bzl

```diff
@@ -49,27 +49,38 @@ def com_google_api_gax_java_repositories():
     # a maven artifact.
     # Remove once upgraded to 3.7.0
     _protobuf_version = PROPERTIES["version.com_google_protobuf"]
-    if _protobuf_version == "3.6.1":
-        _protobuf_version = "3.6.1.3"
+    _grpc_version_in_link = "v%s" % _protobuf_version
```

Comment:
Why is this named _grpc_version_in_link?

Reply:
Because it was me doing copy-paste without thinking... Renamed.